### ISSUE TITLE
In scan mode, focus of screen reader move out of the "list of apis" dialog box.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#127]
User experience:
If, in scan mode focus of Screen Reader move out of dialog box, then the user who depends upon Screen Reader for navigation might get affected hence find it difficult to navigate properly on the dialog controls.

Note: User credentials should NOT be included in the bug.

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the List of API and select it.
8. Navigate with the Screen Reader in scan mode to the various controls of the "List of API" dialog box.
9. Verify, whether in scan mode focus of Narrator move out of dialog box or not.

Actual Result:

In scan mode focus of Screen Reader move out of the "list of apis" dialog box. Instead focus of Screen Reader moves to the background elements.
 

Expected Result:

In scan mode focus of Screen Reader should not move out of the "list of apis" dialog box. So, that the Screen Reader user does able to navigate properly on the dialog box controls.

Note: 1. This issue also exist with Chrome+JAWS+NVDA.
          2. This issue exist throughout the application for the dialog box.

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
